program/idl.json

```diff
@@ -5345,6 +5345,70 @@
           }
         ]
       },
+      {
+        "kind": "instructionNode",
+        "name": "initializeNonTransferableMint",
+        "docs": [
+          "Initialize the non transferable extension for the given mint account",
+          "",
+          "Fails if the account has already been initialized, so must be called before `InitializeMint`."
+        ],
+        "optionalAccountStrategy": "programId",
+        "accounts": [
+          {
+            "kind": "instructionAccountNode",
+            "name": "mint",
+            "isWritable": true,
+            "isSigner": false,
+            "isOptional": false,
+            "docs": ["The mint account to initialize."]
+          }
+        ],
+        "arguments": [
+          {
+            "kind": "instructionArgumentNode",
+            "name": "discriminator",
+            "defaultValueStrategy": "omitted",
+            "docs": [],
+            "type": {
+              "kind": "numberTypeNode",
+              "format": "u8",
+              "endian": "le"
+            },
+            "defaultValue": {
+              "kind": "numberValueNode",
+              "number": 32
+            }
+          },
+          {
+            "kind": "instructionArgumentNode",
+            "name": "nonTransferDiscriminator",
+            "defaultValueStrategy": "omitted",
+            "docs": [],
+            "type": {
+              "kind": "numberTypeNode",
+              "format": "u8",
+              "endian": "le"
+            },
+            "defaultValue": {
+              "kind": "numberValueNode",
+              "number": 0
+            }
+          }
```

Comment:
Unlike your previous PR, this is a single instruction, not an extension with multiple instructions inside of it. Therefore, there isn't a second discriminator and this must be removed.

Reply:
Yes removed the second discriminator and regenerated the files again.